sunpy/extern/modest_image.py

```diff
@@ -0,0 +1,351 @@
+"""
+Modification of Chris Beaumont's mpl-modest-image package to allow the use of
```

Comment:
Has this gone upstream/if not is it worth going upstream?

Reply:
That you would have to ask @astrofrog